addons/packages/harbor/2.2.3/test/e2e/harbor_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
```

Comment:
it could be better if we can format this import libraries. For example, group & sort all built-in libraries into one section, and group & sort the others into the other section.

Reply:
These imports are sorted by build-in, thrid-part, repo itself, and dot imports of the ginkgo.